.gitignore

```diff
@@ -0,0 +1,6 @@
+*.tgz
+.idea
+.vscode
+.DS_Store
+/charts/*/charts
+/charts/*/values.md
```

Comment:
`**/*.dec.*`

Reply:
There will probably some encrypted files, this way they are irgnored.

Reply:
Not in this repository 😉

Reply:
Thats right, but i'm sure this will be cloned and maybe pushed somewhere else. Or am i wrong?

Reply:
No, this repo will only ever contain the source code. The released helm charts are used directly, without cloning.

Reply:
That's a point. You're right :D